config_files/docker_run.sh

```diff
@@ -48,14 +62,17 @@ if [ ! -f ./config/settings.inc.php  ]; then
 		done
 
 		echo "\n* Installing PrestaShop, this may take a while ...";
-		if [ $DB_PASSWD = "" ]; then
-			mysqladmin -h $DB_SERVER -P $DB_PORT -u $DB_USER drop $DB_NAME --force 2> /dev/null;
-			mysqladmin -h $DB_SERVER -P $DB_PORT -u $DB_USER create $DB_NAME --force 2> /dev/null;
-		else
-			mysqladmin -h $DB_SERVER -P $DB_PORT -u $DB_USER -p$DB_PASSWD drop $DB_NAME --force 2> /dev/null;
-			mysqladmin -h $DB_SERVER -P $DB_PORT -u $DB_USER -p$DB_PASSWD create $DB_NAME --force 2> /dev/null;
+		if [ $PS_ERASE_MYSQL = 1 ]; then
```

Comment:
I just noticed this issue, the variable name is different from the ENV var declared in the Dockerfile :wink: